.github/workflows/build-wsl.yaml

```diff
@@ -99,6 +99,19 @@ jobs:
         run: |
           mkdir -p ${{ env.workDir }}/wiki/
           git clone https://github.com/ubuntu/wsl.wiki ${{ env.workDir }}/wiki --depth 1
+      - name: Checkout ubuntu-archive-tools
```

Comment:
I would do that way later in the worklow (once we have pushed to the Store IMHO)